optuna/visualization/matplotlib/_utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from optuna.distributions import LogUniformDistribution
+from optuna.trial import FrozenTrial
+from optuna.visualization.matplotlib import _matplotlib_imports
+
+__all__ = ["is_available"]
+
+
+def is_available() -> bool:
+    """Returns whether visualization with `matplotlib` is available or not.
+
+    .. note::
+
+        :mod:`~optuna.visualization` module depends on Matplotlib version 3.0.0 or higher. If a
```

Comment:
`optuna.visualization.plot_xx` does not depend on Matplotlib. Could you update the module path?

```suggestion
        :mod:`~optuna.visualization.matplotlib` module depends on Matplotlib version 3.0.0 or higher. If a
```